bridgev2/database/upgrades/21-disappearing-message-fkey.postgres.sql

```diff
@@ -0,0 +1,2 @@
+-- v21: Add unique index on portal (bridge_id, mxid)
+CREATE UNIQUE INDEX portal_bridge_mxid_idx ON portal (bridge_id, mxid);
```

Comment:
It should be doable with a single new version rather than 2, although I guess either way is fine

If it's 2 versions, this shouldn't have `.postgres.` in the file name and the versions of the second one should be fixed. New migrations also need the `compatible with` tag like the latest revision